### PR TITLE
Fix ProfileTest generate api key test

### PR DESCRIPTION
**Pull request by djansen-redhat**
It appears that there was a race condition with the alert disappearing
and the number of expected background functions.

### CODE REVIEW

**Comment by aeng** [comment]
:+1:

**Comment by djansen-redhat** [comment]
@aeng I encountered another issue while checking this in, so I fixed them together

File: functional-test/src/test/java/org/zanata/feature/language/JoinLanguageTeamTest.java [hunk 1]
```diff
@@ -66,8 +66,9 @@ public void translatorJoinsLanguageTeam() throws Exception {
         assertThat(languagePage.getMemberUsernames())
                 .contains("translator")
                 .as("Translator is a listed member of the pl team");
-        assertThat(hasEmailRule.emailsArrivedWithinTimeout(1, 5,
+        assertThat(hasEmailRule.emailsArrivedWithinTimeout(1, 30,
```
Comment (aeng) [inline functional-test/src/test/java/org/zanata/feature/language/JoinLanguageTeamTest.java]: Any reason its 30 seconds now? will it increase when the jenkins load is heavy?

Comment (djansen-redhat) [inline functional-test/src/test/java/org/zanata/feature/language/JoinLanguageTeamTest.java]: No, it's just a matter of being an external interface (mail) that may, occasionally, take a touch more than 5 seconds.  We wait 20-30 seconds for everything else, so I think this is ok.

**Comment by aeng** [comment]
:+1:

**Comment by djansen-redhat** [comment]
:white_check_mark: Verified by Leeroy Jenkins

**Pull request merged**